### DISCUSSION TITLE
Reduce disk writes

### DISCUSSION THREAD

**Discussion by amigafan2003** [blakeblackshear/frigate#15184] [Ask A Question]
Hi, I have Frigate set up on TrueNas Scale.  Have four Wyze cams via docker-wyze-bridge.

Currently Frigate seems to be writing 10 seconds clips constantly to disk.  

![Capture](https://github.com/user-attachments/assets/5445d38c-422a-4d3e-97fa-1ee41aeca10f)

Is there any way of reducing the amount of writes?  I thought the following config would only write to disk if motion was detected but that doesn't appear to be the case:-

![Capture](https://github.com/user-attachments/assets/272d56d0-bb7a-4403-b993-19961942a5c9)

**Comment by NickM-27** [comment]
What is your docker config? Do you have /tmp/cache mapped to tmpfs?

Also, have you watched the motion tuner to make sure a timestamp or something isn't causing motion to be detected?

**Reply by amigafan2003** [reply]
Tmpfs as below
![Capture](https://github.com/user-attachments/assets/082992bc-7c39-41de-8c21-71ff6403c293)

History shows fairly large gaps with no motion
![Capture2](https://github.com/user-attachments/assets/272fd149-a2ed-4689-a6d1-ab59158efd47)

**Reply by NickM-27** [reply]
that would just be one camera

**Reply by amigafan2003** [reply]
That's from the review pane > motion > all four cameras

**Reply by NickM-27** [reply]
Gotcha, I still wouldn't go off of that directly. A camera's timestamp might not be enough motion to really show up on the timeline. I would double check the motion tuners to be sure things are working correctly.

**Comment by amigafan2003** [comment]
Ok thank you.  I have set a motion mask for the timestamps.

So as far as config goes, it should be doing what I think it should be doing (i.e. only writing segments with detected motion to disk)?

**Reply by NickM-27** [reply]
yes

**Reply by amigafan2003** [reply]
Brilliant - i will spend some more time investigating the motion settings.
